### DISCUSSION TITLE
CC: Tweaked 1.108 for 1.19.4 and 1.20.1

### DISCUSSION THREAD

**Discussion by SquidDev** [cc-tweaked/CC-Tweaked#1575] [Announcements]
Hello again! Afraid this is a bit of a boring release post (the last few releases have been pretty simple), but there were a couple of things I wanted to mention, so here we are!

## New features in 1.107 and 1.108
As mentioned, the last few releases haven't had any big new features, so instead I'm going to give a quick run down of some of the highlights:

 - **Support [MoreRed](https://modrinth.com/mod/more-red)'s bundled cables:** We used to support MoreRed way back in 1.16.5, but removed this support when we update to more recent versions of Minecraft. It turns out that MoreRed has been updated again, and so we now support it for 1.20.1!

 - **Split GUI textures into sprite sheets:** The next release of Minecraft (1.20.2) splits several of its sprite sheets into individual textures. Looking at this change, it made sense to do this with several of CC's textures as well, as it makes editing textures (especially the computer borders) much easier.

   This does mean texture packs will need to be updated to the new format. [The commit message](https://github.com/cc-tweaked/CC-Tweaked/commit/53546b9f57d9acaa4cdca14ae00eaf68ce8c50bd) describes the changes in a bit more detail, but I've also [written a small Python script](https://github.com/cc-tweaked/CC-Tweaked/blob/cdc8592aa379c9edb6cc0de1d7512a8807c300b7/tools/update-resources.py) which does this transformation for you.

 - **Remove compression from monitor packets:** Way back in 2020, I made a change to [compress the monitor data](https://github.com/cc-tweaked/CC-Tweaked/commit/e3fab2a8a6d94489fde5f5af807dfc7a21801b6c) when sent over the network. Apparently I'd not realised that vanilla applies its own compression, and due to how I was measuring packet size, most of my numbers were entirely wrong.
 
   I've done some testing with and without our own compression (#1569). Just using vanilla's compression increases packet size by about 30%, it also dramatically reduces the CPU cost of creating monitor packets. As such, I think removing compression is worth it.

   Definitely interested to hear from server owners, to know if this change has a positive or detrimental impact!

## Ending 1.19.4 support
While this is uncharacteristically early for me, I'm planning to phase out 1.19.4 support at the end of this release cycle. Most people on 1.19.4 have migrated to 1.20.1, so this shouldn't affect many people.

As with other versions, major bug fixes will still be backported.

**Comment by SirEdvin** [comment]
Eh, I need to update to 1.108 and I don't even updated to 1.107 :)
